src/Shared/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -485,6 +485,14 @@ protected void AssertSourceMappingsMatchBaseline(RazorCodeDocument codeDocument,
                 continue;
             }
 
+            // See https://github.com/dotnet/razor/issues/10062
+            if (expectedSpan.Contains("<TModel>") || span.FirstAncestorOrSelf<RazorDirectiveSyntax>()?.DirectiveDescriptor?.Directive == "model")
```

Comment:
The way we handle `@model` directives is kind of funky, and I couldn't find an easy way to fix it without doing some major changes. I've filed an issue to track fixing it, and for now we just skip them.